creusot/tests/should_fail/bug/01_resolve_unsoundness.mlcfg

```diff
@@ -131,7 +131,7 @@ module CreusotContracts_Logic_Ops_Impl0_IndexLogic
     type self = s,
     type ShallowModelTy0.shallowModelTy = Seq.seq t
   function index_logic [@inline:trivial] (self : s) (ix : int) : t =
-    Seq.get (ShallowModel0.shallow_model self) ix
+    [#"../../../../../creusot-contracts/src/logic/ops.rs" 16 8 16 31] Seq.get (ShallowModel0.shallow_model self) ix
```

Comment:
A bunch of logic things are gaining their (appropriate) spans, since before they were just invalid offsets. Now that we encode spans more properly we are able to map them back onto their source files!